scripts/editFeed.sh

```diff
@@ -0,0 +1,112 @@
+echo "edit feed's heartbeat trigger by alice"
+setHeartbeatTriggerTx=$($chainlinkCMD setHeartbeatTrigger feedid1 20 --from alice --keyring-backend test --chain-id testchain <<< 'y\n')
+setHeartbeatTriggerTxResp=$(echo "$setHeartbeatTriggerTx" | jq '.height')
+if [ "$setHeartbeatTriggerTxResp" == "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setHeartbeatTriggerTx"
+fi
+
+# eDiT dEvIaTiOn ThReShOlD tRiGgEr
+echo "edit feed's deviation threshold trigger by alice"
+setDeviationThresholdTriggerTx=$($chainlinkCMD setDeviationThresholdTrigger feedid1 30 --from alice --keyring-backend test --chain-id testchain <<< 'y\n')
+setDeviationThresholdTriggerTxResp=$(echo "$setDeviationThresholdTriggerTx" | jq '.height')
+if [ "$setHeartbeatTriggerTxResp" == "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setDeviationThresholdTriggerTx"
+fi
+
+# eDiT fEeD rEwArD
+echo "edit feed's reward by alice"
+setFeedRewardTx=$($chainlinkCMD setFeedReward feedid1 40 --from alice --keyring-backend test --chain-id testchain <<< 'y\n')
+setFeedRewardTxResp=$(echo "$setFeedRewardTx" | jq '.height')
+if [ "$setFeedRewardTxResp" == "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setFeedRewardTx"
+fi
+
+# nOn-FeEd OwMeR cAnNoT eDiT
+# eDiT sUbMiSsIoN cOuNt 
+echo "edit feed's submission count by cerlo"
+setSubmissionCountTx=$($chainlinkCMD setSubmissionCount feedid1 10 --from cerlo --keyring-backend test --chain-id testchain <<< 'y\n')
+setSubmissionCountTxResp=$(echo "$setSubmissionCountTx" | jq '.height')
+if [ "$setSubmissionCountTxResp" != "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setSubmissionCountTx"
+fi
+
+# eDiT hEaRtBeAt TrIgGeR
+echo "edit feed's heartbeat trigger by cerlo"
+setHeartbeatTriggerTx=$($chainlinkCMD setHeartbeatTrigger feedid1 20 --from cerlo --keyring-backend test --chain-id testchain <<< 'y\n')
+setHeartbeatTriggerTxResp=$(echo "$setHeartbeatTriggerTx" | jq '.height')
+if [ "$setHeartbeatTriggerTxResp" != "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setHeartbeatTriggerTx"
+fi
+
+# eDiT dEvIaTiOn ThReShOlD tRiGgEr
+echo "edit feed's deviation threshold trigger by cerlo"
+setDeviationThresholdTriggerTx=$($chainlinkCMD setDeviationThresholdTrigger feedid1 30 --from cerlo --keyring-backend test --chain-id testchain <<< 'y\n')
+setDeviationThresholdTriggerTxResp=$(echo "$setDeviationThresholdTriggerTx" | jq '.height')
+if [ "$setDeviationThresholdTriggerTxResp" != "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setDeviationThresholdTriggerTx"
+fi
+
+# eDiT fEeD rEwArD
+echo "edit feed's reward by cerlo"
+setFeedRewardTx=$($chainlinkCMD setFeedReward feedid1 40 --from cerlo --keyring-backend test --chain-id testchain <<< 'y\n')
+setFeedRewardTxResp=$(echo "$setFeedRewardTx" | jq '.height')
+if [ "$setFeedRewardTxResp" != "\"0\"" ]
+then
+  errorAndExit "Error in submitting feed data #1: $setFeedRewardTx"
+fi
+
```

Comment:
There’s a way to query the feed info at the end to verify the value of new submission count, heartbeat trigger, reward etc... ?

Reply:
woah, didn't see this. but to update per our convo with chainlink yesterday, those verification steps won't be necessary anymore. we will merge!